lib/response/converter.ex

```diff
@@ -0,0 +1,47 @@
+defmodule ExVerticalBooking.Response.Converter do
+
+  def convert({:ok, struct, meta}, list_nodes) do
+    {:ok, wrap(%{}, struct, [], list_nodes), meta}
+  end
+
+  def convert(response, _), do: response
+
+  def wrap(acc, {key, %{}, [value]}, _prev, _list_nodes) when is_binary(value) do
+    Map.put(acc, atomize(key), value)
+  end
+
+  def wrap(acc, {key, args, childs}, prev, list_nodes) do
+    arguments =
+      Enum.reduce(
+        args,
+        %{},
+        fn {key, value}, acc -> Map.put(acc, atomize("@#{key}"), value) end
+      )
```

Comment:
`arguments` is used only if we got `:map` from `get_type_for` method, but calculated for all cases